### PR TITLE
[Blog] A closer look: Setting private members in the editor

### PR DESCRIPTION
**Pull request by manio143**
Let me know if the post is alright :)

### CODE REVIEW

**Comment by VaclavElias** [comment]
I think it loos great! Thank you for the great article. Just some optional updates if you want, also could be done after release

- Is it possible, if it is appropriate for your article to cross link to the previous post https://www.stride3d.net/blog/embracing-open-source-stride-as-an-alternative-to-unity/ as you are saying "..maybe port some games from Unity" maybe we could make any of your wordings a link
  - cross referencing for existing articles is very important if happens it makes a sense
- If possible, you can also link to our existing docs e.g. manual, again if it makes a sense e.g. you were mentioning Scripts, so maybe linking here would be useful if the context is related https://doc.stride3d.net/latest/en/manual/scripts/index.html

Otherwise, you can merge to master.

If you want to see it before release, you can merge from `master` to `staging` and see it here https://stride-website-staging.azurewebsites.net/, otherwise you can merge to `release` right away.

**Review by VaclavElias — APPROVED** [review]

**Comment by VaclavElias** [comment]
@manio143, do you want me to release it or shall I wait for your updates?

**Comment by manio143** [comment]
@VaclavElias Sorry, I didn't have the time to look into the fixes since I published the PR. You can move it forward of course. Thank you.

File: posts/2023-09-21-closer-look-private-members.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+```csharp
+public string MyProperty { get; init; }
+```
+
+### `UnsafeAccessor` in .NET 8
+
+As we're trying to squeeze out more performance and improve the Stride codebase we're looking at rewriting the AssemblyProcessor from Mono.Cecil and manually emitting IL into a Roslyn C# source generator. This will also mean being restricted to whatever the compiler enforces, such as not setting init properties outside of object construction.
+
+Luckily in .NET 8 there's a new performance hack which allows you to access private fields on other classes. You can read about it more in the [proposal](https://github.com/dotnet/runtime/issues/90081) and on the [performance blog](https://devblogs.microsoft.com/dotnet/performance-improvements-in-net-8/#networking-primitives).
```
Comment (manio143) [inline posts/2023-09-21-closer-look-private-members.md:76]: @VaclavElias I just realized I linked the wrong proposal link. Should be this one https://github.com/dotnet/runtime/issues/81741

Comment (VaclavElias) [reply posts/2023-09-21-closer-look-private-members.md:76]: That is ok. I can update it as I am still testing from staging :)

**Pull request merged**